lib/suite.js

```diff
@@ -214,6 +215,7 @@ export default class Suite extends EventEmitter {
             this.port = configuration.port;
             this.base_path = configuration.base_path || '';
             this.read_timeout = configuration.read_timeout || 60000;
+            suiteConfigData.common_headers = configuration.common_headers || [];
```

Comment:
`common_headers` coming from js module should go to `this.commonHeaders` object, see line# 268 in master branch